### DISCUSSION TITLE
ne zamn gelecek airdrop

### DISCUSSION THREAD

**Discussion by tahir0007** [IBAX-io/go-ibax#827] [Soru Cevap - Türkçe]
nasıl verielcek airdrop ve ne zaman gelecek bilgisi olan ?

**Comment by Azizatli21** [comment]
1 Şubat ta otomatik cüzdanlar mi yansiyacak

**Comment by Yusufknk** [comment]
1 Şubat'tan sonra dağıtıma başlanacak ve adresleri yollanacak, ama siz yinede ibax sayfasının redeem kısmından kontrol edin.
